package/niflow/nipype1/examples/inspecting_workflows.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+===========================
+Inspecting workflow results
+===========================
+
+It can be useful to investigate individual nodes' inputs and outputs in a workflow that has been run.
+For example, this can help developers write unit tests for their workflows.
+
+"""
+import os
+
+from nipype.interfaces import utility as niu
+from nipype.interfaces.base import (BaseInterfaceInputSpec, File,
+                                    SimpleInterface, TraitedSpec, traits)
+from nipype.pipeline import engine as pe
+
+# First, let's start by creating a simple interface to use in our workflow.
+# The WriteString interface just writes a string to a file.
```

Comment:
Using strings should produce nicer rendering.

```suggestion
"""
First, let's start by creating a simple interface to use in our workflow.
The WriteString interface just writes a string to a file.
"""
```